src/prompt.ts

```diff
@@ -132,7 +140,7 @@ function normal(options: IPromptConfig, retries = 100): Promise<string> {
 function getPrompt(name: string, type?: string, defaultValue?: string) {
   let prompt = '> '
 
-  if (defaultValue && (type && type === 'mask' || type === 'hide')) {
+  if (defaultValue && type === 'hide') {
```

Comment:
In this case, we'll only obfuscate the default if it's type is `hide`, but I would think we'd also want to do the same for `mask`; otherwise, in the the `mask` case, we'll always show the default value in the prompt leading to security issues.

Reply:
I may have jumped the gun.  I assume your creation of `replacePrompt` does the obfuscating?

Reply:
Yes, check line 91.